channels/candidate-4.2.yaml

```diff
@@ -32,3 +33,4 @@ versions:
 - 4.2.13
 - 4.2.13-s390x
 - 4.2.14
+- 4.2.15
```

Comment:
this is 4.2.16 now, won't be releasing 4.2.15. only reason to have it here would be to assist in migrating off of it, but i wouldn't think we even want that.